packages/pds/src/api/com/atproto/server/getSession.ts

```diff
@@ -14,12 +14,25 @@ export default function (server: Server, ctx: AppContext) {
     }),
     handler: async ({ auth, req }) => {
       if (ctx.entrywayAgent) {
-        return resultPassthru(
-          await ctx.entrywayAgent.com.atproto.server.getSession(
-            undefined,
-            ctx.entrywayPassthruHeaders(req),
-          ),
+        // Allow proxying of dpop bound requests by using service auth instead
+        const headers =
+          req.headers.authorization?.split(' ')[0].toLowerCase() === 'dpop'
+            ? await ctx.entrywayAuthHeaders(
+                req,
+                auth.credentials.did,
+                'com.atproto.server.getSession',
+              )
```

Comment:
Since `com.atproto.server.getSession` is proxied only administratively, we may want to add it to `PROTECTED_METHODS` to avoid other usage: https://github.com/bluesky-social/atproto/blob/f476003709d43b5e2474218cd48a9e1d7ebf3089/packages/pds/src/pipethrough.ts#L493-L497